package.json

```diff
@@ -235,6 +235,7 @@
     "array-flat-polyfill": "^1.0.1",
     "blueimp-load-image": "~2.29.0",
     "classnames": "~2.2.5",
+    "concurrently": "^6.4.0",
```

Comment:
This should go in `devDependencies` this way it does not get bundled in the production bundle (unless it's invoked in the code)